configs/zero123.yaml

```diff
@@ -116,6 +116,7 @@ system:
     wandb:
       enable: false
       project: "threestudio"
+      name: None
```

Comment:
Also, ref_or_zero123 needs to be `accumulate`?

Reply:
Also, higher LR?

Reply:
I didn't want to override the defaults just yet... for now you need to run my shell script (which does a bunch of overrides through the command line)

Reply:
Later, when we test on more objects, if we see that this set of config changes consistently outperforms the current default, we can confidently change the default. ;-)